static/js/emoji_picker.js

```diff
@@ -19,6 +19,22 @@ var search_results = [];
 var section_head_offsets = [];
 
 function get_emoji_categories() {
```

Comment:
I am thinking that we should rename these functions to something more clear. Don't have great suggestions right now but may be something like `default_emoji_categories()` -> `get_all_emoji_categories()` and something similar for `get_emoji_categories()`?